client/reader/tag-stream/header.jsx

```diff
@@ -14,11 +21,43 @@ class TagStreamHeader extends Component {
 		following: PropTypes.bool,
 		onFollowToggle: PropTypes.func,
 		showBack: PropTypes.bool,
+		showSort: PropTypes.bool,
+		sort: PropTypes.string,
+	};
+
+	useRelevanceSort = () => {
+		const sort = 'relevance';
+		recordAction( 'tag_page_clicked_relevance_sort' );
+		this.props.recordReaderTracksEvent &&
+			this.props.recordReaderTracksEvent( 'calypso_reader_clicked_tag_sort', {
+				tag: this.props.encodedTagSlug,
+				sort,
+			} );
+		updateQueryArg( { sort } );
 	};
 
+	useDateSort = () => {
+		const sort = 'date';
+		recordAction( 'tag_page_clicked_date_sort' );
+		this.props.recordReaderTracksEvent &&
+			this.props.recordReaderTracksEvent( 'calypso_reader_clicked_tag_sort', {
+				tag: this.props.encodedTagSlug,
+				sort,
+			} );
+		updateQueryArg( { sort } );
+	};
 	render() {
-		const { title, description, isPlaceholder, showFollow, following, onFollowToggle, showBack } =
-			this.props;
+		const {
+			title,
+			description,
+			isPlaceholder,
+			showFollow,
+			following,
+			onFollowToggle,
+			showBack,
+			showSort,
+		} = this.props;
+		const sortOrder = this.props.sort || 'recent';
```

Comment:
I notice in other places we are using `'date'` for the name of this sort order. is `'recent'` a typo here? 

Side note/discussion, should we use `'relevance'` as the default feed?

Reply:
ah yes good catch